### ISSUE TITLE
update release?

### ISSUE THREAD

**Issue by jkanche** [dask/dask-cloudprovider#409]
Its been a while since the last release. Are there plans to update this on PyPI?

**Comment by jacobtomlinson** [comment]
This is true! There haven't been many commits since the last release, but it would be nice to get those out. The CI seems to be unhappy at the moment and I'm not sure I have time to look at it in the short term. Perhaps someone else has some time to troubleshoot what is going on?
